about.md

```diff
@@ -7,39 +7,47 @@ title: About Natron
+We always welcome and are on the lookout for new contributions to the Natron project, may they be minor bug fixes or big new features. As such, we regularly review the GitHub and accept pull requests. If you are a software developer who is well versed in C++, Qt and graphics programming, and are interested in contributing to the Natron project, please do not hesitate to reach out to us!
 
-Natron is suited to perform 2D compositing tasks. That means it can:
+For instructions on how to build Natron from source, read through [the README document here.](https://github.com/NatronGitHub/Natron#building-and-installing-from-source)
 
-* Chromakey blue and green screens
-* Rotoscope a moving object or person
-* Combine multiple pieces of footage on top of each-other
-* Perform 2D stablization and tracking
-* Warp, crop, blur, and other effects
-* Combine and edit render passes of OpenEXR files
+## FAQ
 
-Natron does *not* include 3D compositing features. You will need to use a 3D package like [Blender](https://www.blender.org/) to render your 3D objects before bringing them into Natron for compositing.
+- #### Is Natron really free?
+  Yes, Natron is free software. Free to use, free to modify, and free to share, due to its open-source GNU GPL license.
 
-### Where can I donate to Natron?
+- #### What can Natron do?
+  Natron is a powerful digital compositor capable of handling a multitude of VFX and post-production tasks. Natron's capabilities include but are not limited to:
+   * Linear node-based workflow
+   * 32bpcc floating-point linear processing pipeline
+   * Industry-standard OIIO file format support
+   * Powerful chroma-keying and matte refinement
+   * Industry-standard rotoscoping and rotopaint
+   * Pixel-perfect tracking and stabilization
+   * Industry-standard OCIO color management
+   * Extensive OpenFX and community plugins support
+   * And much more!
 
-While we appreciate your generosity Natron does not currently accept donations.  Currently all of our contributors have full time jobs and the amount we would likely receive from community donations wouldn't justify the administrative overhead of collecting them.  With that caveat aside, if you are a corporate entity willing to fund development on a larger scale please do reach out!
+  Natron can be used in conjunction with [Blender](https://www.blender.org/) to extend it's capabilities by utilising Blender's 3D environment. This is facilitated by Natron's support for CHAN files.
 
-### Where can I find brand assets for Natron?
+- #### Does the Natron Project accept donations?
+  While we appreciate your generosity, the Natron Project does not currently accept donations.
 
-You can download our logo and wordmark on our [brand assets page](/brand).
+- #### Where can I find brand assets for Natron?
+  If you wish to add Natron to the credits of your project or use the logo for other non-commercial purposes, you can download the logomark and wordmark [here](/brand).
```

Comment:
Links with "here" as their descriptive text aren't as accessible as linking with more descriptive text telling users where the link will go.

```suggestion
  If you wish to add Natron to the credits of your project or use the logo for other non-commercial purposes, you can download the logomark on our [brand assets page](/brand).